Control/public/common/appHeader.js

```diff
@@ -41,6 +41,8 @@ const loginButton = (model) => h('.dropdown', {class: model.accountMenuEnabled ?
     h('p.m3.mv2.text-ellipsis', `Welcome ${model.session.name}`, h('sup', model.session.role)),
     model.session.personid === 0 // anonymous user has id 0
       && h('p.m3.gray-darker', 'You are connected as anonymous, no authentification needed for this application.'),
+    model.checkBrowserNotificationPermissions() &&
+      h('a.menu-item', {onclick: () => model.requestBrowserNotificationPermissions()}, 'Enable notifications'),
```

Comment:
Once the button is clicked, I believe it would be better from a UX point of view to:
* display a web-ui notification to inform the user that browser notifications have been enabled; Or even better a browser-notification
* close the drop-down menu

Reply:
ad 1) Once clicked user is notified by browser, so I guess it's good enough.
![Screenshot 2022-07-29 at 15 17 19](https://user-images.githubusercontent.com/1117662/181768762-9635c111-b15d-448f-a4a2-d4668b49d6d5.png)

ad 2) Yes agreed.